components/sidebar/sidebar_channel/sidebar_direct_channel/index.ts

```diff
@@ -17,17 +22,40 @@ type OwnProps = {
     currentTeamName: string;
 }
 
+/**
+ * Gets the LHS bot icon url for a given botUser.
+ */
+function botIconImageUrl(botUser: UserProfile) {
+    return `${Client4.getBotRoute(botUser.id)}/icon?_=${((botUser as any).bot_last_icon_update || 0)}`;
```

Comment:
That field is part of the User object on the server ([link](https://github.com/mattermost/mattermost-server/blob/master/model/user.go#L90)), so we should add it to the type in mattermost-redux

Reply:
Agreed, outside the scope though. We should address all of these at once.

Reply:
Is there a ticket around for that?

Reply:
I'll make one!

Reply:
https://mattermost.atlassian.net/browse/MM-22356